### ISSUE TITLE
Improve a11y of Tooltip component

### ISSUE THREAD

**Issue by julianguyen** [ifmeorg/ifme#2037]
# Description

From the [axe-core](https://github.com/dequelabs/axe-core), Deque's a11y linter:

```
Issue Description
Ensures ARIA attributes are allowed for an element's role

Element location
div[aria-labelledby="tooltip1dmzoq92nwmo65vx3pc0gg"]
Element source
<div class="tooltipElement" aria-labelledby="tooltip1dmzoq92nwmo65vx3pc0gg">
To solve this issue, you need to...
Fix the following:
ARIA attribute cannot be used, add a role attribute or use a different element: aria-labelledby
```

Let's update this component to use `role="tooltip"`, here's more [info](https://www.w3.org/TR/wai-aria-1.1/#tooltip) about that!

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!
